srcpkgs/ardour/template

```diff
@@ -31,6 +28,13 @@ case "$XBPS_TARGET_MACHINE" in
 	*)        broken="Don't know build target" ;;
 esac
 
+# Upstream deletes older release tarballs from their site and
+# GitHub tarballs are empty: https://tracker.ardour.org/view.php?id=7328
+do_fetch() {
+	git clone --branch ${version} --depth=1 \
+		git://github.com/Ardour/ardour ${pkgname}-${version}
+}
```

Comment:
Did you manage to clone a specific hash? If that doesn't work but the repository isn't huge, we could probably clone the entire thing and then check it out. Since retagging is possible, I'd prefer to have a unique identifier.

Reply:
But they won't retag. From https://tracker.ardour.org/view.php?id=7328:

>git-tag is the very first step in the release procedure. If some issues or regressions are found during the release process, the tag is skipped (e.g. there was no Ardour 5.7 release; 5.7-2 became 5.8)

Reply:
I mean the repository isn't huge, but ... large? Takes around a minute to clone on my computer...

Reply:
I'd rather check if the commit hash is correct?

Reply:
I did it with the clone and checkout now. But *if* they retag, the build will probably fail because their build relies on `git describe`, see https://tracker.ardour.org/view.php?id=7328